sql/catalyst/src/main/scala/org/apache/spark/sql/types/Decimal.scala

```diff
@@ -622,6 +623,9 @@ object Decimal {
     override def toLong(x: Decimal): Long = x.toLong
     override def fromInt(x: Int): Decimal = new Decimal().set(x)
     override def compare(x: Decimal, y: Decimal): Int = x.compare(y)
+    // Added from Scala 2.13; don't override to work in 2.12
+    // TODO revisit once Scala 2.12 support is dropped
```

Comment:
Shall we make this as an IDed TODO?

Reply:
Do you mean it shouldn't be a TODO, or should be recorded differently? I just want to leave something in place so when one day I search for "Scala 2.12" because we're cutting out the 2.12-specific stuff, I find this one :)

Reply:
Oh, I mean to have the official JIRA ID and make this `TODO(SPARK-XXX)` style.
It'll give more visibility to the community and contributors can easily find both here and Apache Spark JIRA.